libraries/AC_SafeRTL/examples/SafeRTL_performance_test/SafeRTL_test.h

```diff
@@ -0,0 +1,426 @@
+#include <vector>
+
+#include <AP_BoardConfig/AP_BoardConfig.h>
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Math/AP_Math.h>
+#include <AC_SafeRTL/AC_SafeRTL.h>
+
+// vectors defined below:
+// test_path_before
+// test_path_after_adding
+// test_path_after_simplifying
+// test_path_after_pruning
+// test_path_complete
+
+// assume that any point without a comment should be kept
+std::vector<Vector3f> test_path_before {
```

Comment:
This one is purely for the library example sketch, this is not used in the flight code. I think I just did it this way because it was easy to get a `.size()` method.